### DISCUSSION TITLE
Questions about training data for poses

### DISCUSSION THREAD

**Discussion by AndyLonly** [lllyasviel/ControlNet#132] [Q&A]
I have currently completed the initial training of the SD2.1 canny model and am working on the training of Pose.

I currently have an accurate annotated training data and a large amount of unannotated data,I currently intend to use only the full body of unlabelled data to auto-labelling, and I'm wondering if this has serious implications for practical use after training? Other than that auto-labeling seems to be less accurate, sometimes even a mess, and I was wondering if there is a high percentage of this in your dataset?

less accurate:
![2944418](https://user-images.githubusercontent.com/15814794/220472791-35b81e25-bcb5-48c3-a022-b4ee0ceecd83.jpg)
![2674260](https://user-images.githubusercontent.com/15814794/220472856-a9bc3958-eae5-4fb3-946c-09c0fc039532.jpg)
mess:
![2845803](https://user-images.githubusercontent.com/15814794/220474000-6294d746-5ebc-4b5e-a78b-9c4e779d0fc4.png)
![3639933](https://user-images.githubusercontent.com/15814794/220474085-8c72ba50-dce7-4c63-ace9-5a11ade2507e.jpg)

PS: this pose estimator is for Illustrated character

**Comment by geroldmeisinger** [comment]
I collect information about "promptless training" and "prompt dropping" here https://civitai.com/articles/2078#heading-3617 -> Q: Why does a control net need captions or why would you drop captions?

you might also consider using automated caption generation like BLIP. there might even be specialized ones for characters https://huggingface.co/docs/transformers/model_doc/blip-2
